src/reducers/analysis.js

```diff
@@ -170,19 +172,27 @@ const parseLookupTable = (data) => {
     parsed=results;
   }
   });
-      
+
   return parsed.data;
-}    
+}
 
-const tableHasError = (lookuptable) =>{    
+const tableHasError = (lookuptable) =>{
   let data = parseLookupTable(lookuptable);
   let textOk = !(lookuptable == 'Enter table' || lookuptable == '')
   let parseOk = (data.length > 0 && Object.keys(data[0]).length > 0)
  if (textOk && parseOk ){
     return false;
   } else {
     return true;
-  } 
+  }
+}
+
+const gmfVersionWarnings = (module) => {
+  if(module.gmf_version === undefined || module.gmf_version < CURRENT_GMF_VERSION) {
+    return [{stateName: 'None', message: 'Module is using an older or undeclared version of GMF'}];
```

Comment:
Is there currently a way to address this warning?

Reply:
`gmf_version` will now be added whenever any changes are made in the module.